Workbooks/Azure Update Management/Azure Update Management.workbook

```diff
@@ -0,0 +1,668 @@
+            "customWidth": "50",
+            "name": "query - 0",
+            "styleSettings": {
+              "maxWidth": "50"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "Update\r\n| where TimeGenerated>ago(5h) and OSType==\"Linux\"\r\n| summarize hint.strategy=partitioned arg_max(TimeGenerated, UpdateState, Classification) by Computer, SourceComputerId, Product, ProductArch\r\n| where UpdateState=~\"Needed\"\r\n| summarize by Product, ProductArch, Classification, Computer\r\n| summarize count(Classification) by Classification",
+              "size": 2,
+              "title": "Linux Updates Needed by Classification",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{Workspaces}"
+              ],
+              "visualization": "piechart",
+              "chartSettings": {
+                "seriesLabelSettings": [
+                  {
+                    "seriesName": "Others",
+                    "color": "purple"
+                  },
+                  {
+                    "seriesName": "Security Updates",
+                    "color": "redBright"
+                  }
+                ]
+              }
+            },
+            "customWidth": "50",
+            "name": "top five Computers Needing Updates",
+            "styleSettings": {
+              "maxWidth": "50"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "Update\r\n| where TimeGenerated>ago(14h) and OSType!=\"Linux\" and (Optional==false or Classification has \"Critical\" or Classification has \"Security\")\r\n| summarize hint.strategy=partitioned arg_max(TimeGenerated, UpdateState, Classification, Title, KBID, PublishedDate, Approved) by Computer, SourceComputerId, UpdateID\r\n| where UpdateState=~\"Needed\" and Approved!=false\r\n| summarize count() by Title, Classification\r\n| order by count_ desc \r\n| take 10",
+              "size": 0,
+              "title": "Top 10 Pending Windows Updates (by machine count)",
+              "timeContext": {
+                "durationMs": 86400000
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{Workspaces}"
+              ],
+              "gridSettings": {
+                "labelSettings": [
+                  {
+                    "columnId": "Title",
+                    "label": "Update"
+                  },
+                  {
+                    "columnId": "count_",
+                    "label": "Machines"
+                  }
+                ]
+              }
+            },
+            "customWidth": "50",
+            "name": "query - 6"
```

Comment:
nit: can you look through all the name fields and rename the items to something meaningful and descriptive?